website/docs/docs.md

```diff
@@ -165,6 +165,8 @@ As the name implies, `SidebarItem` is an item defined in a Sidebar. There are a
 - [Ref](#ref)
 - [Category](#category)
 
+**Note**: If you would like to pass in custom props to a swizzled sidebar, an optional object called `extra` can be added to any of the items:
```

Comment:
That does not seem good enough to me, it looks like you forgot to add an example after the `:` ?